test/framework/chargeback.go

```diff
@@ -5,10 +5,13 @@ import (
 	restclient "k8s.io/client-go/rest"
 )
 
-const chargebackServiceName = "metering"
+const (
+	chargebackServiceName     = "metering"
+	chargebackServicePortName = "http"
```

Comment:
`http` is a protocol, so I think the name should be changed to something like `chargebackServiceProtoName`

Reply:
This is already what it is, this is just fixing existing things to use the correct port name. Mostly in docs but adjusting tests to use the right one too.